app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :post, class_name: 'Post', foreign_key: :post_id
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
+
+  def update_comment_counter
+    comment.increment!(:comments_counter)
+  end
+end
```

Comment:
- [ ] Dear @Ngufor-emmanuella, As the previous code reviewer mentioned Comments belong to a post, and the counter should be updated for a specific post. Please make sure to use the `post.increment!(:comments_counter)`, that way the counter for the post will be updated.